cmd/install/assets/hypershift_operator.go

```diff
@@ -140,29 +140,29 @@ func (o HyperShiftOperatorDeployment) Build() *appsv1.Deployment {
 			},
 		},
 	}
-
-	args = append(args,
-		"--oidc-storage-provider-s3-bucket-name="+o.OIDCBucketName,
-		"--oidc-storage-provider-s3-region="+o.OIDCBucketRegion,
-		"--oidc-storage-provider-s3-credentials=/etc/oidc-storage-provider-s3-creds/"+o.OIDCStorageProviderS3SecretKey,
-	)
-	oidcVolumeMount = []corev1.VolumeMount{
-		{
-			Name:      "oidc-storage-provider-s3-creds",
-			MountPath: "/etc/oidc-storage-provider-s3-creds",
-		},
-	}
-	oidcVolumeCred = []corev1.Volume{
-		{
-			Name: "oidc-storage-provider-s3-creds",
-			VolumeSource: corev1.VolumeSource{
-				Secret: &corev1.SecretVolumeSource{
-					SecretName: o.OIDCStorageProviderS3Secret.Name,
+	if len(o.OIDCBucketName) > 0 && len(o.OIDCBucketRegion) > 0 && len(o.OIDCStorageProviderS3SecretKey) > 0 {
```

Comment:
Is this check also guaranty that `o.OIDCStorageProviderS3Secret` is not empty?
User can add `--oidc-storage-provider-s3-bucket-name`, `--oidc-storage-provider-s3-region` and `--oidc-storage-provider-s3-secret-key` flags but not to include one of `--oidc-storage-provider-s3-credentials` or `--oidc-storage-provider-s3-secret` flags
This scenario might also cause to error?

Reply:
Yes please add a Secret check here as well